.svglintrc.js

```diff
@@ -3,12 +3,15 @@ const fs = require('fs');
 const data = require("./_data/simple-icons.json");
 const { htmlFriendlyToTitle } = require("./scripts/utils.js");
 const getBounds = require("svg-path-bounding-box");
+const parsePath = require("svgpath/lib/path_parse");
 
 const titleRegexp = /(.+) icon$/;
 const svgRegexp = /^<svg( [^\s]*=".*"){3}><title>.*<\/title><path d=".*"\/><\/svg>\r?\n?$/;
 
 const iconSize = 24;
 const iconFloatPrecision = 3;
+const iconMinFloatPrecision = 2;
+const iconMaxFloatPrecision = 5;
```

Comment:
Maybe this variables can be called `iconMinFloatPrecisionAverage` and `iconMaxFloatPrecisionAverage`?